image/templates/helm/stackrox-secured-cluster/templates/_init.tpl.htpl

```diff
@@ -268,7 +268,22 @@
 {{/*
    Local Scanner v4 Indexer setup.
   */}}
+
 {{ $scannerV4Cfg := $._rox.scannerV4 }}
+
+{{/* In order to establish a smooth and automated activation of Scanner v4 in situations where Scanner v2 is
+     already activated, we let `scanner.disable: false` have the implicit side-effect of
+    `scannerV4.disable: false` if the latter has not been explicitly set. */}}
+{{- if eq $.Chart.Name "stackrox-secured-cluster-services" -}}
+  {{- if kindIs "invalid" $scannerV4Cfg.disable -}}
+    {{- if not $._rox.scanner.disable -}}
```

Comment:
is it possible to simplify this to `{{- $_ = set $scannerV4Cfg "disable" $._rox.scanner.disable -}}`?